authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.1 on 2018-08-29 07:38
```

Comment:
Is this migration file needed to make travis ci run?

Reply:
yes it is needed @WNjihia, that migration is created by default when you `python manage.py migrate`

Reply:
👍